src/almanack/reporting/report.py

```diff
@@ -0,0 +1,33 @@
+"""
+This module creates entropy reports
+"""
+
+from typing import Dict
+
+
+def repo_entropy_report(data: Dict[str, any]) -> str:
+    """
+    Returns the formatted entropy report as a string.
+
+    Args:
+        data (Dict[str, any]): Dictionary with the entropy data.
+
+    Returns:
+        str: Formatted entropy report.
+    """
+
+    border = "=" * 50
+    separator = "-" * 50
+    title = "Entropy Report"
+
+    # Format the report
+    report_content = f"""
+    {border}
+    {title:^50}
+    {border}
+    Repository Path: {data['repo_path']}
+    Total Repository Normalized Entropy: {data['total_normalized_entropy']:.4f}
+    {separator}
+    {border}
+    """
```

Comment:
Would this also add tabs into the output (for example, lines 25-31)? It's possible they're removed before standard-out, but wanted to double check just in case it impacts the formatting at all.

Reply:
I believe that the format is maintained with standard-out. I have done testing to see the format on the test repos, and everything seemed to align well!